### ISSUE TITLE
New overlapping YouTube UI makes plugin button unusable

### ISSUE THREAD

**Issue by Avadaime** [DeveloperMDCM/Youtube-tools-extension#39]
Home page:
![photo_2024-12-12_15-05-50](https://github.com/user-attachments/assets/a0b90428-1789-4120-b3cb-6ae3c7972a2f)

YouTube Studio:

![photo_2024-12-12_15-10-13](https://github.com/user-attachments/assets/017868c9-2700-447b-9af2-15d47803b9e5)

**Comment by Avadaime** [comment]
Yeah, that's totally not your fault of anything, just another useless YT update that's breaking UI XD.
Appreciate your feedback :)

**Comment by DeveloperMDCM** [comment]
update to 2.3.0
